README.rst

```diff
@@ -1,36 +1,21 @@
 .. image:: https://img.shields.io/badge/%E2%9D%A4-code%20of%20conduct-blue.svg
-    :target: https://github.com/mozilla/addons-server/blob/master/CODE_OF_CONDUCT.md
+    :target: https://github.com/thundernest/addons-server/blob/master/CODE_OF_CONDUCT.md
     :alt: Code of conduct
 
-.. image:: https://travis-ci.org/mozilla/addons-server.svg?branch=master
-    :target: https://travis-ci.org/mozilla/addons-server
-
-.. image:: https://pyup.io/repos/github/mozilla/addons-server/shield.svg
-    :target: https://pyup.io/repos/github/mozilla/addons-server/
-    :alt: Updates
-
-.. image:: https://codecov.io/gh/mozilla/addons-server/branch/master/graph/badge.svg
-    :target: https://codecov.io/gh/mozilla/addons-server
-
-.. image:: https://pyup.io/repos/github/mozilla/addons-server/python-3-shield.svg
-    :target: https://pyup.io/repos/github/mozilla/addons-server/
-    :alt: Python 3
-
 
 Addons-Server
 =============
 
-Welcome to the Addons Server repository!  Please feel free to visit the web page of the current project hosted on `addons.mozilla.org`_. If you want to install it follow our guide located in `install docs`_.  We'd love your help!  You can come talk to us on `irc.mozilla.org #amo`_ if you have any questions.
+Welcome to the Addons Server repository! Please feel free to visit the web page of the current project hosted on `addons.thunderbird.net`_. If you want to install it follow our guide located in `install docs`_.
 
-Please report bugs here: https://github.com/mozilla/addons/issues or https://github.com/mozilla/addons-server/issues
-You can access the AMO dev environment at https://addons-dev.allizom.org/ and the AMO stage environment at https://addons.allizom.org/
+Please report bugs here: https://github.com/thundernest/addons-server/issues
```

Comment:
I think this still makes sense to do, given the confusion here and there;) It may just need updating the URLs throughout now:

```suggestion
Please report bugs here: https://github.com/thunderbird/addons-server/issues
```

… in all the places. (They are redirected, but still…)